src/import-export/import-main-matcher.c

```diff
@@ -82,9 +82,9 @@ enum downloaded_cols
     NUM_DOWNLOADED_COLS
 };
 
-#define COLOR_RED    "brown1"
-#define COLOR_YELLOW "gold"
-#define COLOR_GREEN  "DarkSeaGreen1"
+#define COLOR_RED_CLASS    "intervention-required"
+#define COLOR_YELLOW_CLASS "intervention-probably-required"
+#define COLOR_GREEN_CLASS  "intervention-not-required"
```

Comment:
The values are good (intervention-xyz). Now the same should still happen with the parameter itself. COLOR_RED_CLASS etc is meaningless in the code as well.